### ISSUE TITLE
Accessing processed data values of body parts (Exporting output data to csv or other formats)

### ISSUE THREAD

**Issue by rishabhchandak** [DeepLabCut/DeepLabCut#17]
Hello,

Is there a file that is saving the coordinates of the tracked 'body parts' that I can then export and analyze elsewhere? After running all the steps I end up with a video with overlaid predicted positions of the body parts, but I was wondering if this information is stored somewhere as a CSV or some other format. 

Thanks!

**Comment by AlexEMG** [comment]
I just pushed an update that allows you to save the data also as csv file. Just set the variable: _storedata_as_csv_ to _True_ in _myconfig_analysis.py_ to do this. 

Please also note that the [MultiIndex Panda Arrays](http://pandas.pydata.org/pandas-docs/stable/advanced.html) that we use for holding the pose data as well as the scorer & bodypart information can be exported in many other ways, as indicated in  http://pandas.pydata.org/pandas-docs/version/0.23/api.html#id12

**Comment by rishabhchandak** [comment]
Thank you!!

**Comment by maipbui** [comment]
Hi @AlexEMG,

Would you please be more specific where I can access to the file "myconfig_analysis.py"? Thank you.

**Comment by AlexEMG** [comment]
Hi, this thread is for DLC 1.0 I guess. So the file would be here: https://github.com/AlexEMG/DeepLabCut/releases/tag/1.11

In the new version you can export csv by: 
https://forum.image.sc/t/creating-csv-files-without-re-analyzing/25231/2

and of course during analysis with:
deeplabcut.analyze_videos(config,videos,**save_as_csv=True**)

**Comment by maipbui** [comment]
Thanks a lot! I got the csv file.

**Comment by 3xMoha** [comment]
Hello.

I am using DeepLabCut in a mice tracking experiment. 
I analyzed some videos and got the CSV with the option  **save_as_csv=True**  But there is something I don't understand:

In the generated video, there are predicted points from the beginning of the video (from frame 0) but the CVS doesn't show the X and Y values for any of the body parts in the first 134 frames and also there are a lot of missing values.

I understand some times the points are not predicted or shown in the video (which could be translated as a missing value) but there are far more missing values in the CSV than in the generated video (I checked watching the video by frames and comparing with the data on the CSV)

Also, I tried to get the positions with the pickle files.  But it looks like the pickle file, for some reason, only has the position of one of the mice (I am working on a multi animal project, 2 mice) 

Is there a way to get all the data points as they show in the video from both mice?

Thanks a lot!
